src/doc/src/reference/unstable.md

```diff
@@ -936,7 +936,7 @@ timeout = 300  # in seconds
 * Tracking Issue: [10519](https://github.com/rust-lang/cargo/issues/10519)
 * RFC: [#3231](https://github.com/rust-lang/rfcs/pull/3231)
 
-Add support for Cargo to authenticate the user to registries without sending secrets over the network.
+The `-Z asymmetric-token` flag tells Cargo to authenticate the user to registries without sending secrets over the network.
```

Comment:
```suggestion
The `-Z asymmetric-token` flag enables the `cargo:paseto` credential provider which allows Cargo to authenticate to registries without sending secrets over the network.
```

Reply:
Looks awesome. Thanks for the suggestion :)